test/data/testvib.py

```diff
@@ -130,7 +130,7 @@ def testenthalpy(self):
 
     def testentropy(self):
          """Is the entropy reasonable"""
-         self.assertAlmostEqual(0.03601749, self.data.entropy, self.entropy_places)
+         self.assertAlmostEqual(0.00012080325339594164, self.data.entropy, self.entropy_places)

```

Comment:
Aren't we rounding too soon with `entropy_places = 3` (line 116)?

Reply:
There are uneven rounding places for entropy across programs. Gamess for instance is tested against 6 decimal places.

Reply:
What do you recommend we do instead?

Reply:
As long as the numbers are reliable, we could attempt to be as strict as possible: maximum number of common digits. (The figures I came up with were calculated independently so I believe they are reliable.)

Another possibility would be to set an energy/entropy cut, e.g. 1e-3 hartree (1e-4 hartree) ensures an error smaller than 0.63 kcal/mol (0.063 kcal/mol).

Entropy cuts need then to be calculated separately: by using a cut of 1e-3 hartree/K, we are accepting errors around 187 kcal/mol at 298.15 K! We need 1e-6 hartree/K (1e-7 hartree/K) for errors less than 0.2 kcal/mol (0.02 kcal/mol).

 I prefer 1e-4 hartree and 1e-7 hartree/K.

Reply:
OK then, do you want to make the default for `entropy_places` more stringent and loosen it for select packages?

Reply:
Yes, just to be sure that the parsing is working within a precise but reasonable energy deviation.